server/adaptors/pplDatasource.ts

```diff
@@ -43,21 +42,11 @@ export class PPLDataSource {
         } else {
           record[cur.name] = row[i];
         }
-
-        if (cur.name &&
-            cur.name === 'timestamp' &&
-            cur.type &&
-            cur.type === 'timestamp' &&
-            !hasTimestamp
-          ) {
-            hasTimestamp = true;
-        }
       }
 
       data.push(record);
     });
-    pplRes['jsonData'] = data;
-    pplRes['hasTimestamp'] = hasTimestamp;
+    this.pplDataSource['jsonData'] = data;
```

Comment:
What are the parameters of the `pplDataSource` object, how come `jsonData` is being called with bracket notation vs. dot notation? Should we add a Type or Interface for it?

Reply:
The pplDataSource is just the response object returned from opensearch with schema-to-value mapping added. I'll add interface for it.